### ISSUE TITLE
Howdy Mr.

### ISSUE THREAD

**Issue by coolgithubaccount** [Douggem/Dedougger#1]
Hi, I was initially going to try to contact you on Twitter seeing as you're somewhat active on there. However, the moment after I made an account it was locked requiring some phone verification b.s. To cut to the chase: I have copies of most of the videos from your old YouTube channel that got terminated. I sent you an E-Mail but now realize that there's about a 99% probability it got sent to spam because it was from a newly made E-Mail address. Anyways, hopefully this reaches your notifications/inbox since it'll ping you from GitHub. Drop me your Discord username and we can coordinate some way for me to send you everything.

**Comment by Douggem** [comment]
Yeah your other message did indeed go straight to spam lol.  I replied to
that, but I'll reply here as well: my discord is Douggem#6804
